src/Microsoft.VisualStudio.ProjectSystem.Managed/Microsoft.VisualStudio.ProjectSystem.Managed.csproj

```diff
@@ -95,6 +96,14 @@
       <Generator>MSBuild:GenerateRuleSourceFromXaml</Generator>
       <SubType>Designer</SubType>
     </XamlPropertyRule>
+    <XamlPropertyRule Include="ProjectSystem\Rules\General.BrowseObject.xaml">
```

Comment:
Can you follow the same pattern for the other rules where we add a partial class and mark it with "ExcludeFromCodeCoverage"? We don't cover properties.